### ISSUE TITLE
 AttributeError: module 'keras.api._v2.keras.applications.imagenet_utils' has no attribute 'correct_pad'

### ISSUE THREAD

**Issue by mikechen66** [keras-team/keras-io#1530]
mobilevit.ipynb

AttributeError                            Traceback (most recent call last)
Cell In[4], line 42
     38     return keras.Model(inputs, outputs)
     41 # AttributeError: module 'keras.applications.imagenet_utils' has no attribute 'correct_pad'
---> 42 mobilevit_xxs = create_mobilevit()
     43 mobilevit_xxs.summary()

Cell In[4], line 10, in create_mobilevit(num_classes)
      6 x = inverted_residual_block(
      7     x, expanded_channels=16 * expansion_factor, output_channels=16
      8 )
      9 # Downsampling with MV2 block.
---> 10 x = inverted_residual_block(
     11     x, expanded_channels=16 * expansion_factor, output_channels=24, strides=2
     12 )
     13 x = inverted_residual_block(
     14     x, expanded_channels=24 * expansion_factor, output_channels=24
     15 )
     16 x = inverted_residual_block(
     17     x, expanded_channels=24 * expansion_factor, output_channels=24
     18 )

Cell In[3], line 16, in inverted_residual_block(x, expanded_channels, output_channels, strides)
     13 m = tf.nn.swish(m)
     14 if strides == 2:
     15     # AttributeError: module 'keras.applications.imagenet_utils' has no attribute 'correct_pad'
---> 16     m = layers.ZeroPadding2D(padding=imagenet_utils.correct_pad(m, 3))(m)
     17 m = layers.DepthwiseConv2D(
     18     3, strides=strides, padding="same" if strides == 1 else "valid", use_bias=False
     19 )(m)
     20 m = layers.BatchNormalization()(m)

AttributeError: module 'keras.api._v2.keras.applications.imagenet_utils' has no attribute 'correct_pad'

**Comment by divyashreepathihalli** [comment]
@mikechen66 can you provide a repro for your issue?

**Comment by SuryanarayanaY** [comment]
@mikechen66 ,

I hope you are referring to [mobilevit](https://keras.io/examples/vision/mobilevit/) tutorial.I have replicated the issue reported and attached [gist](https://colab.sandbox.google.com/gist/SuryanarayanaY/359a68752c4bd5d9b7c919ff67560ad1/mobilevit-kerasio-1530.ipynb) for reference. We will have a look into it.

**Comment by SuryanarayanaY** [comment]
Hi @mikechen66 ,

In TF2.13 there are some shufflings happened and also a part of making keras as multi backend. Our Dev team have used a package **namex** for better safegaurd of some of private utilities.

I have corrected the `import` to make it workable and attached [gist](https://colab.sandbox.google.com/gist/SuryanarayanaY/bae80ee6e340f279637c2fe64fe6e49e/mobilevit-kerasio-1530_fixed-tf2-13v.ipynb#scrollTo=lRImjYQIOfBX) here for reference. Basically I changed the import as below that make it works.

`from keras.src.applications import imagenet_utils
`

 I may raise a PR for changing the same in the tutorial. Thank you!
